### ISSUE TITLE
USIS16K split

### ISSUE THREAD

**Issue by peanut699** [LinHong-HIT/USIS16K#1]
Thank you for your fantastic work. I noticed that in your paper, you mentioned that "yielding 11,306 samples for the training set, 3,230 for the validation set, and 1,615 for the test set." However, after downloading the dataset, I found that the number of images in the validation and test sets appears to follow a 1:2 ratio, which is the opposite of what was described in the paper. I would like to ask how you actually split the USIS16K dataset. Thank you.

**Comment by peanut699** [comment]
The author is aware of the issue and will re-upload the dataset in the coming days. Please refer to the dataset split described in the paper. The issue is now closed.
